src/intern/intern-saucelabs.js

```diff
@@ -0,0 +1,34 @@
+define([
+	'./intern'
+], function (intern) {
+
+	intern.capabilities = {
+		project: '<%- appName %>',
+		name: '<%- appName %>',
+		fixSessionCapabilities: false
+	};
+
+	intern.environments = [
+		{ browserName: 'internet explorer', version: [ '10.0', '11.0' ], platform: 'Windows 7' },
+		// { browserName: 'microsoftedge', platform: 'Windows 10' },
+		{ browserName: 'firefox', version: '49', platform: 'Windows 10' },
+		{ browserName: 'chrome', platform: 'Windows 10' },
+		// { browserName: 'safari', version: '9', platform: 'OS X 10.11' },
```

Comment:
but these configs are hidden from the consumer, so I'm not sure what benefit this will have.